modules/nf-core/bracken/bracken/tests/main.nf.test

```diff
@@ -4,12 +4,6 @@ nextflow_process {
     script "../main.nf"
     process "BRACKEN_BRACKEN"
 
-    tag "modules"
-    tag "modules_nfcore"
-    tag "bracken"
-    tag "bracken/bracken"
-    tag "kraken2/kraken2"
-    tag "untar"
```

Comment:
I thought I had deleted that in a different PR already.
Did we put it back when we made our master -> dev sync?

Reply:
Yeah, not sure